homeassistant/components/traccar_server/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""The Traccar Server integration."""
+from __future__ import annotations
+
+from pytraccar import ApiClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+    Platform,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import (
+    CONF_CUSTOM_ATTRIBUTES,
+    CONF_EVENTS,
+    CONF_MAX_ACCURACY,
+    CONF_SKIP_ACCURACY_FILTER_FOR,
+    DOMAIN,
+)
+from .coordinator import TraccarServerCoordinator
+
+PLATFORMS: list[Platform] = [Platform.DEVICE_TRACKER]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Traccar Server from a config entry."""
+    coordinator = TraccarServerCoordinator(
+        hass=hass,
+        client=ApiClient(
+            client_session=async_get_clientsession(hass),
+            host=entry.data[CONF_HOST],
+            port=entry.data[CONF_PORT],
+            username=entry.data[CONF_USERNAME],
+            password=entry.data[CONF_PASSWORD],
+            ssl=entry.data[CONF_SSL],
+            verify_ssl=entry.data[CONF_VERIFY_SSL],
+        ),
+        events=entry.options.get(CONF_EVENTS, []),
+        max_accuracy=entry.options.get(CONF_MAX_ACCURACY, 0.0),
+        skip_accuracy_filter_for=entry.options.get(CONF_SKIP_ACCURACY_FILTER_FOR, []),
+        custom_attributes=entry.options.get(CONF_CUSTOM_ATTRIBUTES, []),
+    )
+
+    await coordinator.async_config_entry_first_refresh()
+
+    hass.data.setdefault(DOMAIN, {})
+    hass.data[DOMAIN][entry.entry_id] = coordinator
```

Comment:
```suggestion
    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
```

Reply:
This is as provided in the scaffolding <https://github.com/home-assistant/core/blob/dev/script/scaffold/templates/config_flow/integration/__init__.py#L18-L22>